source/release-notes.txt

```diff
@@ -375,6 +375,10 @@ New Features:
 
   - Node version 14
 
+Supported Platforms:
```

Comment:
Consider linking to the install considerations

Reply:
Done.